e2e/core-flows.e2e.ts

```
@@ -1636,6 +1636,63 @@ test("connection test-message errors inherit the configured editor accent", asyn
+
+    await expect(editor).toBeVisible();
+    await expect(editor.getByRole("img", { name: "NovelAI style plate preview" })).toBeVisible();
+    expect(errors).toEqual([]);
+  } catch (error) {
+    testFailure = error;
+    throw error;
+  } finally {
+    if (connectionId) {
+      try {
+        const deletionResponse = await page.request.delete(`/api/connections/${connectionId}`);
+        if (!deletionResponse.ok()) throw new Error(`Connection cleanup failed with ${deletionResponse.status()}`);
```

Biome (2.5.6)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)